server/routes/bookRoute.js

```diff
@@ -9,8 +9,20 @@ bookRoute.post('/', Validate.addBook, BookController.addBook);
 bookRoute.get('/', BookController.getAllBooks);
 bookRoute.get('/:id', Validate.id, BookController.getSpecificBook);
 bookRoute.delete('/:id', Validate.id, BookController.deleteBook);
-bookRoute.post('/request', 
-Authenticate.isLoggedIn, Validate.requestBook, BookController.requestBook)
+bookRoute.post(
+  '/request',
+  Authenticate.isLoggedIn,
+  Validate.requestBook,
+  BookController.requestBook
+);
+bookRoute.post(
+  '/recieve/:damaged',
```

Comment:
Why is damaged in the `params` and not the `body`?

Reply:
No strict reason, I'll put it in the body.
I just like adding things that are either true or false in the request parameter